test/integrationtests/LorisIntegrationTest.class.inc

```diff
@@ -203,5 +203,39 @@ abstract class LorisIntegrationTest extends PHPUnit_Framework_TestCase
         }
     }
 
+    /**
+     * Helper function to set user permissions. The function works by deleting
+     * all current permissions for the user then adding the specified permissions
+     * to the TestUser.
+     *
+     * @param array $permissions List of permissions codes to add TestUser
+     *
+     * @return none
+     */
+    function setupPermissions($permissions) {
+        $this->DB->delete("user_perm_rel", array("UserID" => '999990'));
+        foreach ($permissions as $value) {
+            $prepare = $this->DB->prepare(
+                            "INSERT INTO user_perm_rel
+                                SELECT 999990, PermID
+                                    FROM permissions
+                                    WHERE code=:perm"
+                        );
+            $this->DB->execute($prepare, array("perm" => $value));
+        }
+    }
+
+    /**
+     * Helper function to reset user permissions. The function works by adding
+     * all permissions to the TestUser.
```

Comment:
Should point out that it just assumes the test user has to "restore" all permissions since that's the default state of the user, but it doesn't actually verify or store the previous state.